examples/falcon_intel_indicators/README.md

```diff
@@ -0,0 +1,21 @@
+# Falcon Threat Intelligence Indicators
+
+[Falcon X](https://www.crowdstrike.com/endpoint-security-products/falcon-x-threat-intelligence/)
+
+This page represents a stand-alone tool that uses Falcon X Threat Intelligence API to query the Intel Indicators and outputs those vulnerabilities in JSON format to the stdout. This tool can be used together with JSON parsing tools like `jq` in order to build reports of your liking.
+
+## Installation
+
+```
+go get github.com/crowdstrike/gofalcon/examples/falcon_intel_indicators
+```
+
+## Exemplary Usage
+
+Interactive Run:
+```
+$ FALCON_CLIENT_ID="abc" FALCON_CLIENT_SECRET="XYZ" FALCON_CLOUD=us-1 \
+      falcon_intel_indicators --filter="type:'url'+malicious_confidence:'Medium'"  | jq length
```

Comment:
`type:'url'+malicious_confidence:'Medium'` this seems special.... don't suppose there is a way to make this easier or cleaner?

Reply:
It is example just an example of a filter. Will improve the documentation to make that clear.

Reply:
Thanks... was just wondering if there is a better coding solution to create a better customer experience... but that's probably something to think about for another time.